content/blog/_posts/2021-02-fluvio-cloud.md

```diff
@@ -10,158 +10,140 @@ url: /blog/2021/02/announcing-fluvio-cloud-platform
+
+There are tons of reasons why Fluvio is awesome but this is a post about using
+Fluvio Cloud. :)
+
+[Rust's FFI]: https://doc.rust-lang.org/nomicon/ffi.html#calling-foreign-functions
+[Fluvio CLI]: /docs/getting-started/
+[node-js client]: https://github.com/infinyon/fluvio-client-node
+
+# Using The Fluvio Cloud Platform
+
+We've written plenty of [Fluvio tutorials](/tutorials) but we'd like to show
+how Fluvio cloud acts the same as if you were using it locally or if you host
+it yourself.
+
+## Setup
+Setting up the cloud is very straight forward as mentioned in the introduction
+but we'll reiterate those steps anyway. Once you've verified your account (link
+        in the email), a fluvio instance will be automatically provisioned for
+you. Then to make use of it, just do
+```bash
+fluvio cloud login
+```
+This will propt you for the credentials you used to sign up in the web
+form. It will then store this as a profile setting. You can see this setting by
+doing:
+```bash
+fluvio profile view
+```
+
+If you've already got a local instance of fluvio running, you can easily switch
+the instance the client is using by doing:
+```bash
+fluvio profile switch cloud
+```
+
+## Producing and Consuming a datastream
+
+On a fresh Fluvio instance, you'll need to create some topics:
+```bash
+fluvio topic create hello-fluvio-cloud
+```
+
+Getting data in and out of the cloud is just as easy as all our other tutorials:
+
+In one terminal do:
+```bash
+fluvio consume hello-fluvio-cloud
+```
+
+and in another terminal do:
+
+```bash
+fluvio produce hello-fluvio-cloud
+```
+
+The produce command listens to `stdin` and sends it to fluvio cloud on the
+specified topic. In our case, the topic is `hello-fluvio-cloud`. Now, type
+`Hello fluvio cloud!` (or whatever you'd like) into the producer terminal.
+
+Your two terminals should now look like:
+```bash
+$ fluvio produce hello-fluvio-cloud
+Hello fluvio cloud!
+Ok!
+```
+
+```bash
+$ fluvio consume hello-fluvio-cloud
+Hello fluvio cloud!
+```
+
+Similar to using the commandline, using the [rust
+client](https://crates.io/crates/fluvio) and [nodejs
+client](https://www.npmjs.com/package/@fluvio/client) requires no addional
+steps to use Fluvio Cloud.
+
+We'll leave out the project setup steps but if you have:
+
+```javascript
+const fluvio = await Fluvio.connect();
+const producer = await fluvio.topicProducer('hello-fluvio-cloud');
+await producer.sendRecord("Hello Fluvio Cloud! 🎉");
+```
+
+and
+```rust
+let consumer = fluvio::consumer("hello-fluvio-cloud", 0).await?;
+let mut stream = consumer.stream(Offset::beginning()).await?;
+
+while let Some(Ok(record)) = stream.next().await {
+    let string = String::from_utf8_lossy(&record.as_ref());
+    println!("Got record: {}", string);
+}
+```
+
+And run the rust version in one terminal and the node version in the other
+terminal, the rust consumer will print:
+```
+Got Record: Hello Fluvio Cloud! 🎉
+```
+
+
```

Comment:
### Fluvio Platform Highlights

.....

Reply:
* **Declarative Management**: Fluvio provisioning system allows operators to declare desired state and the system take care of the rest. No resource are available, no worries, resource are listed in progress until appropriate resources are provisioned. System is also designed to self heal for temporary outages.

* **Low Latency**: Fluvio takes advantage of Rust’s async runtime to improve system performance and achieve low latency. It utilizes all available cores and interacts directly with hardware I/O. Unlike other streaming platforms where garbage collection adds unpredictable latency, Fluvio latency is constant and predictable.

* **Low Memory Footprint**: Fluvio can be deployed in as little as 100Mb of memory. The deployment is highly optimized machine code and it does not require an intermediary virtual machine. Fluvio can be run on devices such as the Raspberry Pi and are suitable for IOT devices running at the edge.

* **Built-in Retention**: Fluvio uses long-lived immutable storage to persist data streams. Each data stream is copied to multiple servers and retained for hours or years. Fluvio clients have built-in failure detection and switch-over mechanisms to minimize downtime. Fluvio’s retention mechanism allows clients to come and go without concerns over data loss.

* **Guaranteed Message Ordering**: Fluvio guarantees partition-level message ordering. Messages are stored and forwarded to consumers in the order they are received from the producers. Order guarantee is a critical requirement in many stateful systems.

* **Cloud Native by Design**: Fluvio is designed to work natively with Kubernetes. It uses Helm charts for installation, CRDs for provisioning, and Operators to interact with the KV store.

* **Developer Friendly**: Fluvio offers native language bindings in many modern programming languages such as Rust and Node. Future release will gradually introduce additional languages such as Python, Java, and Swift.

* **Powerful CLI:** Fluvio offers a powerful CLI that controls virtually all aspects of the system: installation, provisioning, data streaming, and monitoring. All-in-one CLI to manage all your clusters whether they are installed locally, in your data center, or in a public cloud.

Reply:
Feel free to shorten the sentences.